templates/templates/base.html

```diff
@@ -2,7 +2,7 @@
 <!doctype html>
 
 {# Release versions - update as appropriate #}
-{% with latest_release_name="ArtfulAardvark" latest_release='17.10' latest_release_with_point="17.10.1" latest_release_eol='July 2018' lts_release_name="XenialXerus" lts_release="16.04" lts_release_full='16.04 LTS' lts_release_with_point="16.04.4" lts_release_full_with_point='16.04.4 <abbr title="Long-term support">LTS</abbr>' lts_release_eol='April 2021' lts_openstack_version="Mitaka" latest_openstack_version="Pike" %}
+{% with latest_release_name="BionicBeaver" latest_release='18.04' latest_release_with_point="18.04.1" latest_release_eol='July 2019' lts_release_name="Zesty Zapus" lts_release="17.04" lts_release_full='17.04 LTS' lts_release_with_point="17.04.1" lts_release_full_with_point='17.04.1 <abbr title="Long-term support">LTS</abbr>' lts_release_eol='April 2022' lts_openstack_version="Mitaka" latest_openstack_version="Pike" %}
```

Comment:
There is a new OpenStack version being released for 18.10. Therefore, `lts_openstack_version` and `latest_openstack_version`should be Queens.